### PR TITLE
doc/rbd: Fix switched `accessMode` claim supports

### PR DESCRIPTION
**Pull request by giovanism**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by varshar16** [comment]
jenkins test docs

File: doc/rbd/rbd-kubernetes.rst [hunk 1]
```diff
@@ -207,9 +207,9 @@ provision a `PersistentVolume`, which would be backed by a Ceph block image.
 An optional `volumeMode` can be included to select between a mounted file system
 (default) or raw block device-based volume.
 
-Using `ceph-csi`, specifying `Filesystem` for `volumeMode` can support both
-`ReadWriteOnce` and `ReadOnlyMany` `accessMode` claims, and specifying `Block`
-for `volumeMode` can support `ReadWriteOnce`, `ReadWriteMany`, and
+Using `ceph-csi`, specifying `Filesystem` for `volumeMode` can support all
+`ReadWriteOnce`, `ReadOnlyMany`, and `ReadWriteMany` `accessMode` claims, and
```
Comment (dillaman) [inline doc/rbd/rbd-kubernetes.rst:211]: The RBD `ceph-csi` driver does not support `ReadWriteMany` under `Filesystem` volume mode -- under `Block` mode `ReadWriteMany` is supported.

Comment (giovanism) [reply doc/rbd/rbd-kubernetes.rst:211]: Yes, as it should be.

Comment (dillaman) [reply doc/rbd/rbd-kubernetes.rst:211]: ... am I missing something? Your change is incorrect -- you are saying that somehow ext4 or XFS file systems are supported on RBD block devices under ReadWriteMany?

Comment (giovanism) [reply doc/rbd/rbd-kubernetes.rst:211]: Sorry, You're right. I just take a look at ceph-csi support matrix for rbd and cephfs. Previously, I had confused cephfs ceph-csi with rbd ceph-csi on filesystem volume mode. I'm closing this PR.

**Pull request closed without merging**